packages/main/src/ListItemBase.js

```diff
@@ -36,6 +36,18 @@ const metadata = {
 			noAttribute: true,
 		},
 
+		/**
+		 * Defines whether <code>ui5-li</code> is in disabled state.
+		 * <br><br>
+		 * <b>Note:</b> A disabled <code>ui5-li</code> is noninteractive.
+		 * @type {boolean}
+		 * @defaultvalue false
```

Comment:
Can you add since tag - 1.0.0-rc.12 here and for the Option

Reply:
Done